product_docs/docs/migration_toolkit/55/07_invoking_mtk/mtk_command_options_in_file/executing_migration_with_txt.mdx

```diff
@@ -4,31 +4,31 @@ navTitle: "Executing a migration with the options file"
 deepToC: true
 ---
 
-After you have created the options file, execute the migration command by referencing the options file:
+After you create the options file, reference it when executing the migration command:
 
 ```shell
-./runMTK.sh -optionsFile <example.options>
+./runMTK.sh -optionsFile <options_textfile> <schema_name>
 ```
 
 !!!note
-   Provide the full path for `<example.options>` if you are running the MTK command from a different folder than where you have stored the file.
+   Provide the full path for `<options_textfile>` if you're running the Migration Toolkit command from a different folder from where you stored the file.
 !!!
 
-You can specify all options in the file, or use a mix of command line and options file to specify the migration parameters.
+You can specify all options in the file or use a mix of the command line and the options file to specify the migration parameters.
 
 ## Provide the scope for the schema migration 
 
-You must specify the scope of the schemas to be migrated.
+Specify the scope of the schemas to migrate:
 
--   If you want to migrate all schemas: Add the `-allSchemas` option in the command line or add `allSchemas` in your options file. Regardless of which method you use, this option must be the last parameter.
+-   If you want to migrate all schemas, add the `-allSchemas` option at the command line or add `allSchemas` in your options file. For both methods, this option must be the last parameter.
 
--   If you want to specify one or several schemas: Specify the schemas you want to migrate in the command line with no preceding option. They have to be the last parameter in the command line.
+-   If you want to specify one or several schemas, specify the schemas you want to migrate at the command line with no preceding option. Schema specifications must be the last parameter at the command line.
```

Comment:
This statement is true, but hides the fact that several schemas can actually be specified. 

It would be better to say something like the following,

>  If you want to specify a subset of schemas (one or more), specify the schemas as comma separated list of schema names you want to migrate at the command line with no preceding option. Schema specifications must be the last parameter at the command line.